store/rootmulti/proof.go

```diff
@@ -1,31 +1,10 @@
 package rootmulti
 
 import (
-	"bytes"
-	"errors"
-	"fmt"
-
 	"github.com/tendermint/iavl"
 	"github.com/tendermint/tendermint/crypto/merkle"
 )
 
-// MultiStoreProof defines a collection of store proofs in a multi-store
```

Comment:
Note to review: This was for handling the serialized `StoreInfos[]` as a proof, which is now removed in favor of a merkle commitment (which scales much better for dozens or more of substores)